tests/integration/modules/test_vmc_vcenter_stats.py

```diff
@@ -0,0 +1,98 @@
+"""
+    Integration Tests for vmc_vcenter_stats execution module
+"""
+import pytest
+import requests
+from saltext.vmware.utils import vmc_vcenter_request
+
+
+@pytest.fixture
+def common_data(vmc_vcenter_admin_connect):
+    hostname, username, password, verify_ssl, cert = vmc_vcenter_admin_connect
+    data = {
+        "hostname": hostname,
+        "username": username,
+        "password": password,
+        "verify_ssl": verify_ssl,
+        "cert": cert,
+    }
+    yield data
+
+
+@pytest.fixture
+def request_headers(common_data):
+    return vmc_vcenter_request.get_headers(
+        common_data["hostname"], common_data["username"], common_data["password"]
```

Comment:
**question** why not just do 

```
def request_headers(vmc_vcenter_admin_connect):
   return vmc_vcenter_request.get_headers(
        common_data["hostname"], common_data["username"], common_data["password"],
        )
```        

Giving things a different name doesn't magically make them a different thing. It's still the same data. In fact:

```
def request_headers(vmc_config):
    common_data = vmc_config["vmc_vcenter_admin_connect"]
```

would work exactly the same as this, and eliminate two fixtures.

Reply:
updated